src/main/java/com/minecolonies/entity/ai/citizen/lumberjack/EntityAIWorkLumberjack.java

```diff
@@ -374,40 +367,40 @@ private boolean plantSapling(BlockPos location)
 
         int saplingSlot = findSaplingSlot();
 
-        if(saplingSlot != -1)
+        if (saplingSlot != -1)
         {
             ItemStack stack = getInventory().getStackInSlot(saplingSlot);
-            Block block = ((ItemBlock) stack.getItem()).getBlock();
+            Block     block = ((ItemBlock) stack.getItem()).getBlock();
             worker.setHeldItem(saplingSlot);
 
             placeSaplings(saplingSlot, stack, block);
 
             world.playSoundEffect((float) location.getX() + HALF_BLOCK_OFFSET,
-                    (float) location.getY() + HALF_BLOCK_OFFSET,
-                    (float) location.getZ() + HALF_BLOCK_OFFSET,
-                    block.stepSound.getBreakSound(),
-                    block.stepSound.getVolume(),
-                    block.stepSound.getFrequency());
+                                  (float) location.getY() + HALF_BLOCK_OFFSET,
+                                  (float) location.getZ() + HALF_BLOCK_OFFSET,
+                                  block.stepSound.getBreakSound(),
+                                  block.stepSound.getVolume(),
+                                  block.stepSound.getFrequency());
             worker.swingItem();
         }
 
-        if(job.tree.getStumpLocations().isEmpty() || timeWaited >= MAX_WAITING_TIME)
+        if (job.tree.getStumpLocations().isEmpty() || timeWaited >= MAX_WAITING_TIME)
         {
             timeWaited = 0;
-            setDelay(10);
+            setDelay(TIMEOUT_DELAY);
             return true;
         }
         return false;
     }
 
     private void placeSaplings(int saplingSlot, ItemStack stack, Block block)
     {
-        while(!job.tree.getStumpLocations().isEmpty())
+        while (!job.tree.getStumpLocations().isEmpty())
         {
             BlockPos pos = job.tree.getStumpLocations().get(0);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'pos' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)